### PR TITLE
Update zh-CN language file

### LINKED ISSUE #3397: Demo language should not encourage creating an account on Alpha
sandstorm Demopad
Samstag 1 August 2020 12:00
little hint, after some years  ... tears ... ;)

good stuff on sandstorm,
also after some years looking around
;)

but let me say some little hint:

  the acc crossLogin gh seems doesnt work ...

Create account
Your demo data will be transferred to your new account.
with GitHubwith Googlewith e-mail

gh, gmai both has no action  / reaction

after connecting to sandstorm

dont understand because this usability is so long in the repo
looks not good,

may be somebody thinks here is someone collecting accountdata by connecting gh, goo acc's
 to sandstorm without some need

https://alpha.sandstorm.io/grain/GsTJZGHExhutxEWNF4f9fD/

https://alpha.sandstorm.io/grain/ubzMFNCPre4KsXFyMAHRRT/  

CU
tdiky9ccdj9l@opayq.net

keep healty!
:)

### PR DESCRIPTION
**Pull request by misaka00251**
Previous work: #3236 

 - [x] Updated translation
 - [x] Fix #3397

### CODE REVIEW

**Comment by ocdtrekkie** [comment]
Thanks! We really appreciate i18n work a lot!

**Comment by ocdtrekkie** [comment]
Oops, wrong label 😂

**Comment by misaka00251** [comment]
Well, it's mostly done except the LDAP and SAML part. I'm not using any of that now :(

Also, I didn't translate the `styleguide` part. I can't find that in the admin page, where is it? Or is it hidden somewhere?

**Comment by ocdtrekkie** [comment]
Honestly, I have no idea how to get to the styleguide anymore. I am also not sure why it was part of the codebase instead of the documentation. I wouldn't worry about it much. Thank you for all of your hard work!

File: shell/i18n/zh-CN.i18n.json [hunk 1]
```diff
@@ -219,6 +219,15 @@
             "confirmHint": "下载备份",
             "confirm": "下载备份"
         },
+        "grainCloneButton": {
+            "hint": "克隆沙粒",
+            "text": "克隆",
+            "copyTitle": "Copy of %s"
```
Comment (zenhack) [inline shell/i18n/zh-CN.i18n.json:225]: Looks like you missed `copyTtile`

**Review by zenhack — COMMENTED** [review]
One thing spotted in-line. Thanks for doing this!

Re: the styleguide, it appears not to be linked from anywhere, but served on /styleguide e.g.

https://alpha.sandstorm.io/styleguide

...but I had to grep around in the source for it; I had no idea this page existed previously. It does seem like something that would make more sense as a stand-alone document anyway.

I'm sure there's some history here; maybe @kentonv can enlighten us.

File: shell/i18n/zh-CN.i18n.json (hunk 1 above)
Comment (misaka00251) [reply shell/i18n/zh-CN.i18n.json:225]: Sorry, my bad :( Will fix it ASAP.

**Comment by ocdtrekkie** [comment]
I think it **might've** been so it can outright use the style elements in the Sandstorm codebase, but either way, I'm just sure localization for it is particularly important at this time.

File: shell/i18n/zh-CN.i18n.json (hunk 1 above)
Comment (misaka00251) [reply shell/i18n/zh-CN.i18n.json]: Fixed!

**Comment by kentonv** [comment]
Thanks!

**Pull request merged**